components/project/ProjectSlides.vue

```diff
@@ -81,32 +81,39 @@
       changeSlide(move) {
         this.goTo(this.active + move);
       },
-      goTo(toActive) {
-        toActive = toActive || 0;
+      toActive(index) {
         const min = 0;
         const max = (this.slides.length - 1);
-        toActive = (toActive < min) ? min : toActive;
-        toActive = (toActive > max) ? max : toActive;
+        let to = index || 0;
+        to = (to < min) ? min : to;
+        to = (to > max) ? max : to;
+        return {
+          active: to,
+          id: this.slides[to].id,
+          prev: ((to === min) ? min : (to - 1)),
+          next: ((to === max) ? max : (to + 1)),
+        }
+      },
+      setActive(to) {
+        this.active = to.active;
+        this.prev = to.prev;
+        this.next = to.next;
```

Comment:
Not that I think this is an issue here, but I find myself wondering if you can batch changes in Vue, so you trigger one re-render.